docker/oso-host-monitoring/rhel7/root/config.yml

```diff
@@ -99,6 +99,9 @@
       minute: "*/2"
       job: ops-runner -f -s 15 -n csrc.openshift.node.registry.checks /usr/bin/cron-send-registry-checks
 
+    - name: send openshift-node process Mem usage and Cpu usage
+      minute: "*/2"
+      job: ops-runner -f -s 15 -n csrc.openshift.node.process.usage /usr/bin/cron-send-cpu-mem-process
```

Comment:
the "csrc" you see usually matches the script name.  This one was crom above, cron-send-reg-check (csrc).  This should be updated to be "cscmp"